docusaurus.config.js

```diff
@@ -11,7 +11,7 @@ const config = {
   title: "Clearpath Robotics Documentation",
   url: "https://docs.clearpathrobotics.com",
   baseUrl: "/",
-  onBrokenLinks: "throw",
+  onBrokenLinks: "warn",

```

Comment:
Are we sure we want to change this? I'm ok to change it on the ros-humble branch, but I don't think we would push this change up to development.

Reply:
This is temporary. I think a lot more links will break while we restructure everything. Just need to remember to revert this.

Reply:
Ok sounds good